Add required steps for installing OSS Calico on Openshift v4.16+ on AWS (#1820)
tigera/docs: calico_versioned_docs/version-3.28/getting-started/kubernetes/openshift/installation.mdx

openshift-install create manifests
```

:::note

> If you are using OpenShift **v4.16 or newer** on **AWS**, at this point you need to edit the OpenShift cluster-api manifests
> that configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic in your cluster.

Reviewer: ```suggestion
For OpenShift **v4.16 or newer** on **AWS**, configure AWS security groups to allow BGP, typha and IP-in-IP encapsulation traffic by editing the OpenShift cluster-api manifests
```

Reviewer 2: I also prefer one sentence per line. Much easier to review in git.
